spec/models/post_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  before(:all) do
+    @user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.',
+                        posts_counter: 0)
+  end
+
+  context 'Validations' do
+    it 'checks for title presence' do
+      post = Post.new(author: @user, text: 'This is my first post', likes_counter: 0, comments_counter: 0)
+      expect(post.valid?).to eq false
+    end
+
+    it 'checks for title length' do
+      test_title = 'I' * 260
+      post = Post.new(author: @user, title: test_title, text: 'This is my first post', likes_counter: 0,
+                      comments_counter: 0)
+      expect(post.valid?).to eq false
+    end
+
+    it 'checks if likes_counter is greater or equal to zero' do
+      post = Post.new(author: @user, title: 'Post communication', text: 'This is my first post', likes_counter: -1,
+                      comments_counter: 0)
+      expect(post.valid?).to eq false
+    end
+
```

Comment:
- [ ] please add test to check if like and comment counter are integers

Reply:
Noted.